tests/test_flows.py

```diff
@@ -1327,5 +1327,11 @@ def test_beacon_report_query(self):
     t.start_test('init')
     env.launch_environment_docker(options.unique_id, options.skip_init, options.tag)
 
+    for agent in env.agents:
+        debug("sending dev_set_config to an agent")
+        agent.cmd_reply("dev_set_config,program,map,backhaul,eth")
+
+    time.sleep(7)
```

Comment:
Why 7 seconds and not less?

Reply:
Otherwise it fails on my laptop :)

Reply:
Yes, it is brittle, but it is out of scope of this PR. PPM-354 is created for this purpose.